scripts/forge-scripts/CreateL2ArbSysProposal.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
is there a reason this license is different? if there is, can you remind me? elsewhere we use AGPL3